python/turbine_models/tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+def pytest_addoption(parser):
```

Comment:
How do these tests actually work? It will just run each config so will only test nothing crashes, but won't actually test if anything is functioning correctly will it?

Reply:
run vmfb_comparison just runs both torch and turbine models, then prints the output of both for us to compare. Not really a real test. https://github.com/nod-ai/SHARK-Turbine/pull/217 does add a comparison for SD which we might be able to use similarly